storage/innobase/xtrabackup/doc/source/prerequisites/permissions.rst

```diff
@@ -0,0 +1,134 @@
+
+.. _pxb.privilege:
+
+
+Permissions and Privileges Needed
+================================================================================
+
+For the purposes of this manual, *permissions* refers to the ability of a user to access and perform changes on the relevant parts of the host's filesystem, such as starting or stopping services, or installing software.
+
+*Privileges* refer to the abilities of a database user to perform different kinds of actions on the database server.
```

Comment:
*Privileges* refers to the ability of a database user

Reply:
fixed.